Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -79,6 +79,7 @@ public Vector2 Forward
     private GameObject teammate;
     private Ball ball;
     private Player player;
+    private bool eventSent = false;
```

Comment:
What is this for?

Reply:
By default PlayerMovement will set velocity to 0 constantly if no events are sent/no controller is plugged in for a player. This fucks up the physicstransformview obviously since it's trying to set velocity without sending those events. So use a bool, and only set velocity in player movement if an event was actually sent.